app/views/avo/actions/show.html.erb

```diff
@@ -26,9 +26,9 @@
       <%= form.hidden_field :avo_resource_ids, value: params[:id] || params[:resource_ids], 'data-action-target': 'resourceIds' %>
       <%= form.hidden_field :avo_selected_query, 'data-action-target': 'selectedAllQuery' %>
       <%= form.hidden_field :arguments, value: params[:arguments] %>
-      <% if @action.get_fields.present? %>
+      <% if @fields.present? %>
         <div class="mt-4 -mx-6">
-          <% @action.get_fields.each_with_index do |field, index| %>
+          <% @fields.each_with_index do |field, index| %>
             <%= render field
```

Comment:
Here when initializing the component we should decide if autofocus is on or off, something like:

`.component_for_view(@view).new(..., autofocus: index == 0)`

Reply:
I tried something like this, but it only works for the text field.